cmd/kpod/docker/types.go

```diff
@@ -0,0 +1,271 @@
+package docker
```

Comment:
It was manually imported in buildah, so I did the same here.  I can vendor it but I thought there might be a reason it wasn't vendored in the first place